### PR TITLE
Add configs for CI, fix tests and update README

### PR DESCRIPTION
**Pull request by RJMW**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,11 +7,11 @@
 
 def main():
 
-    if sys.version_info[0] != 3 and sys.version_info[1] <= 7:
```
Comment (ludwigc) [inline setup.py]: Doesn't sys.version_info[1] >= 7 allow python 3.6?

File: setup.py [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
-        sys.exit("Python-3.7.3 is required ")
+    if sys.version_info[0] != 3 and sys.version_info[1] >= 7:
```
Comment (ludwigc) [inline setup.py]: Doesn't sys.version_info[1] >= 7 allow python 3.6?

File: appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+environment:
+  matrix:
+    - PYTHON: "C:\\Python37"
```
Comment (ludwigc) [inline appveyor.yml]: Is this windows specific?

File: setup.py (hunk 2 above)
Comment (RJMW) [reply setup.py]: No 3.7 or higher

File: appveyor.yml (hunk 3 above)
Comment (RJMW) [reply appveyor.yml]: Appveyor is for windows and Travis for unix

**Pull request merged**